frontend/package.json

```diff
@@ -71,6 +71,7 @@
     "@vitejs/plugin-react-swc": "^3.6.0",
     "@vitest/coverage-v8": "^1.3.1",
     "autoprefixer": "^10.4.17",
+    "cypress": "^13.7.2",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-import": "^2.29.1",
```

Comment:
@bcgov-brwang - are we missing a `script` entry in order to launch cypress locally?

Reply:
maybe
`
"test:cypress": "cypress run" 
`
something like that?

Reply:
or `"test:e2e"` ? 🤷